video_xblock/mixins.py

```diff
@@ -54,3 +54,44 @@ def populate_default_values(self, fields_dict):
         for key, value in self.settings.items():
             fields_dict.setdefault(key, value)
         return fields_dict
+
+
+class LocationMixin(XBlock):
```

Comment:
if we need it only in test purposes, can we simply decorate VideoXblock instance and add these methods?
Something like that:
```py
instance = LocationDecorator(VideoXBlock(...))
```
Does it make sense?